### PR TITLE
Frontend Models (persistent logic)

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Still work in progress
- We have the concept of `models` on the frontend. Perhaps ill-named (suggestions welcome), but at the end of the day, a "model" is just a logic that's always there and stores some kind of persistent data. 
- This PR adds `eventPropertiesLogic` and `propertyDefinitionsLogic` into the `models` folder (and renames them to `*Model` as well)
- Loads all models at the app level and only after we have an authorized `user`.
- Creates another model for `api/person/properties` and updates various property filters to use it

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] Frontend/CSS is usable at 320px (iPhone SE) and decent at 360px (most phones)

### CODE REVIEW

File: frontend/src/scenes/App.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { BackTo } from 'lib/components/BackTo'
 import { Papercups } from 'lib/components/Papercups'
 import { appLogicType } from './AppType'
 import { PreflightStatus } from '~/types'
+import { models } from '~/models'
```
Comment (macobo) [inline frontend/src/scenes/App.tsx:20]: Nitpicking: mixing relative `./AppType`, tilde absolute `~/models` and absolute `lib/components/Papercups` imports is confusing.

File: frontend/src/scenes/App.tsx [hunk 2]
```diff
@@ -60,7 +61,22 @@ export const appLogic = kea<appLogicType<PreflightStatus>>({
 
 export function App(): JSX.Element | null {
     const { showApp, showingDelayedSpinner } = useValues(appLogic)
-    return showApp ? <AppScene /> : showingDelayedSpinner ? <SceneLoading /> : null
+    const { user } = useValues(userLogic)
+
+    return showApp ? (
```
Comment (macobo) [inline frontend/src/scenes/App.tsx:66]: Nitpick: This ternary is a bit much now IMO. Worth extracting the first case like this?

```ts
if (showApp) {
  return (
    <>            
      {user ? <Models /> : null}
      <AppScene />
    </>
  )
}

**Review by macobo — APPROVED** [review]
:muscle:

File: frontend/src/scenes/App.tsx (hunk 1 above)
Comment (mariusandra) [reply frontend/src/scenes/App.tsx:20]: Agreed, though I'd like to tackle this another time. Absolute paths for `lib` and `scenes` work without the tilde, everything else requires a tilde. That's confusing and touches a lot of files.

**Pull request merged**